spg/Animal/lib/Horse.pm

```diff
@@ -46,18 +61,28 @@ sub get_type {
 
 =head2 get_voice
 
+A horse sound
+
 =cut
 
 sub get_voice {
   return "neigh";
 }
 
+=head2 DESTROY
+
+Doing the right thing and recycling
+
+=cut
+

```

Comment:
Moose has its own notion of a [destructor](http://search.cpan.org/~ether/Moose-2.1603/lib/Moose/Manual/Concepts.pod#Destructor), you'll probably need to use that instead of overriding `DESTROY` so that Moose can do its own cleanup.